### ISSUE TITLE
kebab-case filenames and .yml property names.

### ISSUE THREAD

**Issue by pavlovcik** [ubiquity/ubiquibot#296]
1. All file names should be renamed to kebab case.
2. All of the `.yml` property names should also be in kebab case.

              I feel like we should eventually refactor all of the properties to be kebab case in this file.

_Originally posted by @pavlovcik in https://github.com/ubiquity/bounty-bot/pull/282#discussion_r1184663864_

**Comment by Steveantor** [comment]
Why not `camelCase` for .ts files and `kebab-case` for yml files?
I don't see any universal typescript filename convention.

**Comment by pavlovcik** [comment]
Git is not case sensitive, but macOS is. I think its irresponsible to rely on casing for file names as it causes inconsistencies. 

This is an issue I faced first hand in the ubiquity dollar repo some months ago. 

Kebab case is a best practice for web and files being served over servers, which is partially relevant here (static assets and compiled javascript files.) 

I think the steel man argument for camelcase is that it saves a couple of seconds when refactoring code in vscode to move functions to a new file. The cons here are the potential git and server issues which I faced first hand in the other repository. 

This is why we should use kebab case for file names.
